Armadillo/src/main/java/com/scribd/armadillo/Reducer.kt

```diff
@@ -175,14 +175,29 @@ internal object Reducer {
                             isSeeking = action.isSeeking,
                             seekTarget = action.seekPositionTarget)
                 } else MediaControlState()
-
+                val progress = if (action.seekPositionTarget != null) {
```

Comment:
For most of these, I think `seekPositionTarget` should really be non-null. It's only used in `SeekAction` to resolve the end of a seek. Can you make it non-null for all the other actions where it's never actually null?

Reply:
Did not notice that. Sure, I can make that change.